### DISCUSSION TITLE
Joining the Weird Network

### DISCUSSION THREAD

**Discussion by erlend-sh** [muni-town/weird#155] [General]
How does one join the Weird network as a personal page?

1. Build page with Weird-app
2. Integrate with Weird (Leaf)
   - API
   - snippets
3. Opt-in for (semantic) scraping in robots.txt

Related:
- https://github.com/muni-town/weird/discussions/220

**Comment by zicklag** [comment]
For option 3 are you thinking Weird would be something of a world-wide web crawler or something like that?

How would we know which sites to crawl, and who would pay for storage in that scenario? Or would you tell Weird specifically to crawl your site?

**Reply by erlend-sh** [reply]
It’s a half-baked idea, but I was imagining something like:

- submit just your site’s URL to our directory via a ‘submit site by URL’ tab there
- prove that you want your site scraped by having a rel=me, robots.txt or whatever’s easiest.
